### PR TITLE
Added touch-icon support for a majority of devices

### PR DESCRIPTION
**Pull request by chrisheninger**
Based off Mathias Bynens’ research:
https://mathiasbynens.be/notes/touch-icons

I added the png files to the same directory as the favicon.ico file.
I used the adorable voatgoat from https://github.com/voat/voat .

Hope this is an appropriate solution to https://voat.co/v/ideasforvoat/comments/75897 .

### CODE REVIEW

**Comment by voat** [comment]
Good work, thank you! :+1:

**Comment by chrisheninger** [comment]
Awesome! I pulled it up on live to check it out and noticed that the icon files aren't linked, did the 9 different PNGs get uploaded to the root directory? It's expecting them at: https://voat.co/apple-touch-icon-180x180-precomposed.png

**Comment by voat** [comment]
No, they did not upload to root. I just moved them to graphics/icons, this should be live and working now. Can you confirm?

**Comment by voat** [comment]
Another small thing: the goat is too tiny to be useful on Firefox, I think we should stick to the "upvoat button icon" from https://voat.co/favicon.ico.

Here's how it looks on my end:

small icon

File: Whoaverse/Whoaverse/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -114,6 +114,25 @@
 
     <link rel='shortcut icon' href="/favicon.ico" type="image/x-icon" />
 
+    <!-- For Chrome for Android: -->
+    <link rel="icon" sizes="192x192" href="touch-icon-192x192.png">
```
Comment (quantumwannabe) [inline Whoaverse/Whoaverse/Views/Shared/_Layout.cshtml]: I believe this is overriding the existing favicon.ico (the blue arrow) as you did not add a new favicon.ico file. This thread has some complaints about it: https://voat.co/v/Voat/comments/76885. 

The more modern way to do icons for Chrome for Android is to make a web app manifest file like this: http://updates.html5rocks.com/2014/11/Support-for-installable-web-apps-with-webapp-manifest-in-chrome-38-for-Android

**Comment by chrisheninger** [comment]
Shoot, that's strange. It seems to be displaying the favicon correctly in Chrome/Safari, but Firefox is pulling the touch goat icon as the favicon.

I popped it open on a dev server, and figured out that by simply moving the favicon.ico link file below the touch icons fixes the problem in Firefox. I'll submit a PR with the fix, my apologies!

**Pull request merged**